packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -554,14 +556,59 @@ private ConnectionStatusCallback requestConnectionStatusCallback(final Callback
         };
     }
 
+    @ReactMethod
+    public void showAndroidSoftKeyboard(Integer delayMillis) {
+        Activity currentActivity = mReactContext.getCurrentActivity();
+        if (currentActivity != null) {
+            View currentFocusedView = currentActivity.getCurrentFocus();
+            if (currentFocusedView != null) {
+                // Cancel any previously scheduled Runnable
+                if (mKeyboardRunnable != null) {
+                    currentActivity.getWindow().getDecorView().removeCallbacks(mKeyboardRunnable);
+                }
+
+                final Runnable showTheKeyboardNow = new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            View activeFocusedView = currentActivity.getCurrentFocus();
+                            if (activeFocusedView != null && currentActivity.getWindow().getDecorView().isShown()) {
+                                InputMethodManager imm =
+                                    (InputMethodManager) mReactContext.getSystemService(Context.INPUT_METHOD_SERVICE);
+                                imm.showSoftInput(activeFocusedView, InputMethodManager.SHOW_IMPLICIT);
+                            }
+                        } catch (Exception e) {
+                            // Noop
+                        }
+                    }
+                };
+
+                // Schedule the keyboard showing with a delay
+                if (currentFocusedView.hasWindowFocus()) {
+                    currentActivity.getWindow().getDecorView().postDelayed(showTheKeyboardNow, delayMillis);
```

Comment:
I debugged the app and couldn't manage to reproduce a case where this part would be invoked. I presume we'd need it in case this function is invoked when Aztec view is focused, is this accurate?

Reply:
Nice catch! I've just tested as well and it is not being called in any of the cases when opening the different modals. I will remove that part 👍